media-video/pipewire/pipewire-0.3.23.ebuild

```diff
@@ -0,0 +1,207 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson
+
+if [[ ${PV} == 9999 ]]; then
+	EGIT_REPO_URI="https://github.com/PipeWire/pipewire.git"
+	EGIT_BRANCH="work"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/PipeWire/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~arm ~arm64 ~ppc ~ppc64 ~x86"
+fi
+
+DESCRIPTION="Multimedia processing graphs"
+HOMEPAGE="https://pipewire.org/"
+
+LICENSE="LGPL-2.1+"
+SLOT="0/0.3"
+IUSE="alsa aac bluetooth debug doc extra ffmpeg-spa gstreamer jack ldac pipewire-alsa +pipewire-jack systemd test udev v4l vulkan-spa"
```

Comment:
again, I'm not a dev, but maybe you don't want to preselect pipewire-jack here for everyone?

Reply:
Other than building the emulation support of an already very short build, it has absolutely no drawbacks - no dependencies, no surprising change to system behavior, nothing. In fact, per Gentoo guidelines it very well could be always enabled without a USE flag to toggle it (and I was/am half expecting developers to require just that).

Reply:
There are several `IUSE` in this list that do not tick any box inside `src_configure()` which makes them invalid.
Example: If I have `media-libs/fdk-aac` installed, the pipewire build results in linking to this package despite having `IUSE="-aac"` set.

Reply:
It should be working now with the latest commit. Let me know if I missed anything.